### PR TITLE
[FIX] Added code to display 'No results found' message in directory search

### LINKED ISSUE #1547: 'No results found' message on Directory search view not displayed
- Your Rocket.Chat Experimental app version: 4.2.1.9999999
- Your Rocket.Chat server version: 2.5.0-develop open.rocket.chat
- Device (or Simulator) you're running with: Android P

Similar as #1533 . On the directory view also, we don't get 'no results found' message.

ezgif com-crop(3)

### PR DESCRIPTION
**Pull request by aashaypalliwar**
@RocketChat/ReactNative

Closes #1547

### CODE REVIEW

**Comment by m1-key** [comment]
Just a suggestion , if you display the message in all possible language supported by this app , then it would be quite good .

**Comment by aashaypalliwar** [comment]
Yes, I will do that !

File: app/views/DirectoryView/index.js [hunk 1]
```diff
@@ -213,6 +213,10 @@ class DirectoryView extends React.Component {
 		);
 	}
 
+	renderEmptyList = () => (
+		<Text style={styles.emptyList}>{I18n.t('no_search_result_found')}</Text>
```
Comment (djorkaeffalexandre) [inline app/views/DirectoryView/index.js]: Need themed text, like:
`style={[styles.emptyList, {  color: themes[theme].titleText }]}`.

File: app/i18n/locales/en.js [hunk 2]
```diff
@@ -264,6 +264,7 @@ export default {
 	No_messages_yet: 'No messages yet',
 	No_Reactions: 'No Reactions',
 	No_Read_Receipts: 'No Read Receipts',
+	no_search_result_found: 'No search results found',
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/en.js]: You can use `No_results_found` and not need create it.

File: app/views/DirectoryView/styles.js [hunk 3]
```diff
@@ -66,6 +66,13 @@ export default StyleSheet.create({
 		marginTop: 2,
 		...sharedStyles.textRegular
 	},
+	emptyList: {
+		fontSize: 20,
```
Comment (djorkaeffalexandre) [inline app/views/DirectoryView/styles.js]: IMO, it's better if you use `fontSize: 14` and `sharedStyles.textBold`. :)

File: app/i18n/locales/en.js (hunk 2 above)
Comment (aashaypalliwar) [reply app/i18n/locales/en.js]: Hi @djorkaeffalexandre , I am sorry, I didn't get you.

File: app/views/DirectoryView/index.js [hunk 4]
```diff
@@ -213,9 +213,13 @@ class DirectoryView extends React.Component {
 		);
 	}
 
-	renderEmptyList = () => (
-		<Text style={styles.emptyList}>{I18n.t('no_search_result_found')}</Text>
-	)
+	renderEmptyList = () => {
+		// eslint-disable-next-line no-unused-vars
+		const { isFederationEnabled, theme } = this.props;
```
Comment (djorkaeffalexandre) [inline app/views/DirectoryView/index.js]: Why `isFederationEnabled`? You can remove it.

**Pull request closed without merging**